backend/src/nodes/nodes/pytorch/convert_to_onnx.py

```diff
@@ -19,17 +20,29 @@
 class ConvertTorchToONNXNode(NodeBase):
     def __init__(self):
         super().__init__()
-        self.description = """Convert a PyTorch model to ONNX."""
-        self.inputs = [ModelInput("PyTorch Model")]
-        self.outputs = [OnnxModelOutput(label="ONNX Model")]
+        self.description = """Convert a PyTorch model to ONNX.
+            Note: fp16 conversion will only work if PyTorch fp16 mode is turned on."""
+        self.inputs = [
+            ModelInput("PyTorch Model"),
+            OnnxFpDropdown(),
+        ]
+        self.outputs = [
+            OnnxModelOutput(label="ONNX Model"),
+            TextOutput("FP Mode", "FpMode::toString(Input1)"),
+        ]
 
         self.category = PyTorchCategory
         self.name = "Convert To ONNX"
         self.icon = "ONNX"
         self.sub = "Utility"
 
-    def run(self, model: PyTorchModel) -> OnnxModel:
+    def run(self, model: PyTorchModel, is_fp16: int = 0) -> Tuple[OnnxModel, str]:
```

Comment:
> `= 0`

Default values don't make sense for input parameters because the input itself determines the default. Please remove this and just pass 0 (or better, a constant, e.g. `FP_MODE_32`) in `convert_to_ncnn.py`.